timber/src/test/java/timber/log/TimberTest.java

```diff
@@ -467,6 +467,38 @@ void run() {
         .hasNoMoreMessages();
   }
 
+  @Test public void isLoggableReturnsTrueIfAtLeastOneTreeIsLoggable() {
+    Timber.plant(new Timber.DebugTree() {
+      @Override protected boolean isLoggable(String tag, int priority) {
+        return "FIRST".equals(tag);
+      }
+    });
+
+    Timber.plant(new Timber.DebugTree() {
```

Comment:
I guess planting just one tree here is enough and hence this one could be removed.